pickly-common/src/main/java/version/Version.java

```diff
@@ -0,0 +1,42 @@
+package version;
+
+public class Version implements Comparable<Version> {
+
+  public static final String MAX_VERSION = "99.99";
+
+  private final int major;
+
+  private final int minor;
+
+  public Version(String version) {
+    String[] tokens = version.split("\\.");
+
+    if (tokens.length != 2) {
+      throw new IllegalArgumentException("Invalid version format: " + version);
+    }
+
+    major = Integer.parseInt(tokens[0]);
+    minor = Integer.parseInt(tokens[1]);
+  }
+
+
+  @Override
+  public int compareTo(Version version) {
+    if (this.major > version.major) {
+      return 1;
+    } else if (this.major < version.major) {
+      return -1;
+    } else if (this.minor > version.minor) {
+      return 1;
+    } else if (this.minor < version.minor) {
+      return -1;
+    } else {
+      return 0;
```

Comment:
다음과 같이 수정하면 코드 이해하기가 더 간결해질 것 같습니다. if else문은 사용할 때 if 조건이 뭐였더라 하고 생각하는데 if문만 사용하면 과거 정보를 기억할 필요 없이 해당 줄만 읽으면 되니까요~
```suggestion
  @Override
  public int compareTo(Version version) {
    if (this.major > version.major) {
      return 1;
    }
    
    if (this.major < version.major) {
    	return -1;
    }
    
    if (this.minor > version.minor) {
    	return 1;
    }
    
    if (this.minor < version.major) {
    	return -1;
    }
    
    return 0;
```

Reply:
오우 제시 감사합니다 수정해놓을게용